### PR TITLE
ZER-9 Create products table

### PR DESCRIPTION
**Pull request by Har4enkoO**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: db/migrate/20210409175743_create_products.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateProducts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :products do |t|
+      t.uuid :uuid
```
Comment (maksymst) [inline db/migrate/20210409175743_create_products.rb:4]: Please add a unique constraint for `uuid`

File: db/migrate/20210409175743_create_products.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+      t.string :title
+      t.integer :product_type_id
```
Comment (maksymst) [inline db/migrate/20210409175743_create_products.rb:6]: Read this and fix your migration https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html#method-i-references

**Review by maksymst — CHANGES_REQUESTED** [review]

File: db/migrate/20210409175743_create_products.rb (hunk 1 above)
Comment (Har4enkoO) [reply db/migrate/20210409175743_create_products.rb:4]: Thanks, added index

File: db/migrate/20210409175743_create_products.rb (hunk 2 above)
Comment (Har4enkoO) [reply db/migrate/20210409175743_create_products.rb]: Thanks, changed type of field.

**Review by maksymst — APPROVED** [review]

**Pull request merged**